### PR TITLE
Audit error messages

### PR DESCRIPTION
**Pull request by sunho**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR adds a verification script that finds out badly capitalized error messages and corrects them.

**Which issue(s) this PR fixes** 
Correct #15863 partly.

**Special notes for your reviewer**:
I have no idea about how to add go script to hack properly. For now, I just placed it in hack/verify-error-msgs/main.go. To run it, you can cd to the dir, go build and run the binary. 

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Comment by sunho** [comment]
I signed it

**Comment by nikhita** [comment]
>  To run it, you can cd to the dir, go build and run the binary.

We probably want a script which does this.

@sunho Thanks for addressing this through an automated way though! :100: 


/cc @cblecker @dims

**Comment by mbohlool** [comment]
cc @logicalhan

File: pkg/cloudprovider/providers/aws/aws_loadbalancer.go [hunk 1]
```diff
@@ -1198,7 +1198,7 @@ func (c *Cloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadBala
 			modifyAttributesRequest.LoadBalancerAttributes = loadBalancerAttributes
 			_, err = c.elb.ModifyLoadBalancerAttributes(modifyAttributesRequest)
 			if err != nil {
-				return nil, fmt.Errorf("Unable to update load balancer attributes during attribute sync: %q", err)
```
Comment (mourya007) [inline pkg/cloudprovider/providers/aws/aws_loadbalancer.go]: I couldn't find any relevancy to replace capital letters from here. Do you have any other reason for fixing this ?

File: hack/verify-error-msg.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+go run "${KUBE_ROOT}/hack/verify-error-msg/main.go" $KUBE_ROOT
```
Comment (mourya007) [inline hack/verify-error-msg.sh]: nit: One new line at the end is necessary.

File: hack/verify-error-msg/BUILD [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    tags = ["automanaged"],
+    visibility = ["//visibility:private"],
+)
+
+filegroup(
+    name = "all-srcs",
+    srcs = [":package-srcs"],
+    tags = ["automanaged"],
+    visibility = ["//visibility:public"],
+)
```
Comment (mourya007) [inline hack/verify-error-msg/BUILD]: ditto

**Pull request closed without merging**